src/main/java/HttpServer/ResponseWriter.java

```diff
@@ -36,4 +39,16 @@ public void write() throws IOException {
         System.out.println("<CRLF>");
         writing.writeLine("");
     }
+
+    private String requestLine(HashMap request) throws IOException {
+        String requestedResource = (String) request.get("URI");
+        boolean found = resources.stream().anyMatch(resource -> requestedResource.matches(resource));
```

Comment:
What exactly is this line doing? I'm having trouble figuring it out just from looking at the code on GitHub.

Reply:
`resources` is a collection of strings. Do any of those strings match the URI that was requested?

Reply:
:white_check_mark: